### ISSUE TITLE
Feature: Auto Refresh

### ISSUE THREAD

**Issue by tgxn** [tgxn/lemmy-modder#106]
**What do you want to see?**
Setting options for auto-refresh:
- header query counts - default 5 mins
- content - default when count changes?
- other content - 1hour

**Additional context**
so that the counts update when i get new things....

could also look at implementing browser notifications when:
- notification recieved
- new report (count increased)
- new approval (count increased)

**Comment by Pdzly** [comment]
This could be easily be implemented with redux-toolkit ( https://redux-toolkit.js.org/rtk-query/usage/queries ) and with automatic polling feature of such queries ( https://redux-toolkit.js.org/rtk-query/usage/polling ) or do you have already something in mind?

**Comment by tgxn** [comment]
yep, exactly. there are currently cache and stale timers that are just set to infinite
https://github.com/tgxn/lemmy-modder/blob/main/src/hooks/useLemmyHttp.js#L42C9-L42C14

mainly need the additional setting ui and then read it to load the value for the correct stale time in each component that uses `useLemmyHttp`

**Comment by Pdzly** [comment]
Ahhh ok, you do the useQuery from  redux. (I think it ) Could be simplified with the redux-toolkit createAPI. But both are good options ;)
